docusign/metadata.go

```diff
@@ -77,3 +80,24 @@ func (c *Connector) GetPostAuthInfo(ctx context.Context) (*common.PostAuthInfo,
 
 	return nil, ErrNoDefaultAccount
 }
+
+func (c *Connector) SetPostAuthInfo(ctx context.Context, info *common.PostAuthInfo) error {
```

Comment:
Sample example of how the setter will fill in the missing gaps.

Docusing will set the BaseURL. First get the catalog, resolve substitution which we now can resolve and lastly set API url.